CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ Code freeze date: YYYY-MM-DD
 
 ### Changed
 
+- `climada.hazard.centroids.centr.union` and `climada.hazard.centroids.centr.append`[#989](https://github.com/CLIMADA-project/climada_python/pull/989). Results: computing windfields is 10 times faster.
+  - `union` does not iterate anymore over `append` but calls directly `append` with the entire list of centroids objects.
+  - `append` concatenate centroids only once, when all centroids have been appended, instead of concatenating after every centroid is appended.
```

Comment:
Please merge these into a single change like
```suggestion
- `Centroids.append` now takes multiple arguments and provides a performance boost when doing so [#989](https://github.com/CLIMADA-project/climada_python/pull/989)
```
You can then add another point to "Fixed" with:
```
- Resolved an issue where windspeed computation was much slower than in Climada v3 [#989](https://github.com/CLIMADA-project/climada_python/pull/989)
```